### DISCUSSION TITLE
Vue support is terrible, it doesn't work out of the box

### DISCUSSION THREAD

**Discussion by jipika** [vuejs/language-tools#1451] [Q&A]
There is no intellisense on the template, nor is there any hint of vue instructions
Want to be able to seamlessly switch between vuE2 and VUe3 projects instead of two extensions

**Comment by kingyue737** [comment]
After adding [@volar-plugins/vetur](https://github.com/johnsoncodehk/volar-plugins/tree/master/packages/vetur), I've uninstalled vetur to avoid disabling either vetur or volar in a workspace.

**Comment by douglasg14b** [comment]
I also get no intellisense or hints in templates with Vue 2.7 and Volar. It's quote frustrating especially because there appears to be a missing overlap area between Vetur & Volar when it comes to trying to migrate from Vue 2 -> Vue 3. 

Unfortunately not addressed with that Vetur plugin :/

**Reply by johnsoncodehk** [reply]
Volar supports TS intellisense in template for Vue 2, please make sure you don't have `"experimentalDisableTemplateSupport": true` in tsconfig.
